src/kv/kvtypes.h

```diff
@@ -5,13 +5,16 @@
 #include "crypto/hash.h"
 
 #include <array>
+#include <chrono>
 #include <functional>
 #include <limits>
 #include <memory>
+#include <unordered_set>
 #include <vector>
 
 namespace kv
 {
+  using ConfigIndex = int64_t;
```

Comment:
This is not a ConfigIndex, and Term also needs a rename.

Reply:
Agreed on SeqNo and View instead of Index (or ConfigIndex) and Term

Reply:
Added as next step in #300